app_dart/lib/src/service/datastore.dart

```diff
@@ -147,4 +151,25 @@ class DatastoreService {
       return previousStatusUpdates.single;
     }
   }
+
+  /// Generate new authorization token for [agent]
+  ///
+  /// The hashed code of token will be returned as a list
+  List<int> refreshAgentAuthToken(Agent agent){
```

Comment:
Nit: space in between close parens and open curly brace.  Does `dartfmt` not do this automatically?

Reply:
Running darfmt to have that space.